app/models/docker_container_wizard_states/exposed_port.rb

```diff
@@ -0,0 +1,16 @@
+module DockerContainerWizardStates
+  class ExposedPort < Parameter
+    ExposedPort.reset_callbacks(:validate)
```

Comment:
Please leave a comment here and in `exposed_port.rb` explaining the reason :smile: